### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by uMartinXu**
Signed-off-by: uMartinXu martin.xu@intel.com

### CODE REVIEW

**Comment by vbedida79** [comment]
+1:+1:

**Review by vbedida79 — COMMENTED** [review]
+1:+1:

File: README.md [hunk 1]
```diff
@@ -5,41 +5,29 @@ This project is currently under active development, and as such, all source code
...
 
-![figure 1. Intel Data Center GPU Driver Container Image Releasing Flow](pics/releasing_flow.jpg)
+Intel data center GPU driver container image is built from [Intel GPU Driver Repository](https://github.com/intel-gpu). It includes:
+-	[Intel® Graphics (i915) Driver Backports for Linux for RHEL](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/redhat/main)
+-	[Intel® Converged Security Engine (CSE) Backports for RHEL-8.6](https://github.com/intel-gpu/intel-gpu-cse-backports/tree/rhel86)
```
Comment (hershpa) [inline README.md:12]: Refer link to main branch for cse and pmt. The rhel86 branch is not being maintained anymore.

File: README.md [hunk 2]
```diff
@@ -5,41 +5,29 @@ This project is currently under active development, and as such, all source code
...
 
-Intel dGPU driver container image is built from [Intel GPU Drivers](https://github.com/intel-gpu) which includes
+To build Intel Data Center GPU driver images and provision the GPU cards on OCP, two scenarios have to be supported.
 
-*	**[Intel GPU i915 driver for RHEL](https://github.com/intel-gpu/intel-gpu-i915-backports)**
-* **[Intel® Converged Security Engine(CSE) driver for RHEL](https://github.com/intel-gpu/intel-gpu-cse-backports)**
-*	**[Intel® Platform Monitoring Technology(PMT) driver for RHEL](https://github.com/intel-gpu/intel-gpu-pmt-backports/)**
-*	**[Intel® GPU firmware](https://github.com/intel-gpu/intel-gpu-firmware)** 
+- **Driver Upgrade Scenario** - There is a new release from Intel GPU Drivers.In this scenario, Intel data center GPU driver Dockerfile needs to be updated to include the new release, Intel data center driver container image will be built and packaged from the Dockerfile. After testing and certifying steps, the new image should be released on Red Hat Ecosystem Catalog.
```
Comment (hershpa) [inline README.md:19]: Add a space between "There is a new release from Intel GPU Drivers." and "In this scenario"

File: README.md (hunk 1 above)
Comment (uMartinXu) [reply README.md:12]: let me submit another PR to record this change.

**Comment by mjkadera** [comment]
Did this recent PR change complete the team review? I don't see a +1 in the comments.

**Pull request merged**